xml/installation-installation-configure_lbaas.xml

```diff
@@ -232,482 +242,363 @@
+| availability_zone_hints |                                                                                                       |
+| availability_zones      | nova                                                                                                  |
+| created_at              | 2019-10-23T13:30:21Z                                                                                  |
+| description             |                                                                                                       |
+| distributed             | False                                                                                                 |
+| external_gateway_info   | {"network_id": "2ed7deca-81ed-45ee-87ce-aeb4f565d3ad", "enable_snat": true, "external_fixed_ips":     |
+|                         | [{"subnet_id": "45d31556-08b8-4b0e-98d8-c09dd986296a", "ip_address": "10.84.57.38"}]}                 |
+| flavor_id               | None                                                                                                  |
+| ha                      | False                                                                                                 |
+| id                      | 1c9949fb-a500-475d-8694-346cf66ebf9a                                                                  |
+| interfaces_info         | [{"subnet_id": "c141858a-a792-4c89-91f0-de4dc4694a7f", "ip_address": "12.12.12.1", "port_id":         |
+|                         | "3d8a8605-dbe5-4fa6-87c3-b351763a9a63"}]                                                              |
+| name                    | lb_router2                                                                                            |
+| project_id              | de095070f242416cb3dc4cd00e3c79f7                                                                      |
+| revision_number         | 3                                                                                                     |
+| routes                  |                                                                                                       |
+| status                  | ACTIVE                                                                                                |
+| tags                    |                                                                                                       |
+| updated_at              | 2019-10-23T13:33:22Z                                                                                  |
++-------------------------+-------------------------------------------------------------------------------------------------------+</screen>
    </step>
    <step>
     <para>
-     Create the first floating IP.
+     Create a security group for testing with unrestricted inbound and outbound traffic (you will have to restrict the traffic later):
```

Comment:
Wrap this text so it is readable - usually around 80 chars

Reply:
wrapped this and a few other spots I noticed... will need to do another sweep as I likely missed a few